src/pages/projectSettings/tabs/PluginsTab/types.ts

```diff
@@ -9,19 +9,17 @@ export interface FormState {
       jiraCustomFields: Array<{
         field: string;
         displayText: string;
-      }> | null;
+      }>;
     };
     useBuildBaron: boolean | null;
-    ticketSearchProjects: Array<{
-      searchProject: string;
-    }> | null;
+    ticketSearchProjects: Array<{ searchProject: string }>;
     ticketCreateProject: {
-      createProject: string | null;
+      createProject: string;
     };
     fileTicketWebhook: {
-      endpoint: string | null;
-      secret: string | null;
-    } | null;
```

Comment:
Same as other type changes; due to the way `PluginTab/transformers.ts` is implemented, these fields are never actually `null`.